frontend/app/controllers/Joiner.scala

```diff
@@ -96,8 +96,7 @@ trait Joiner extends Controller with ActivityTracking with LazyLogging {
         case Tier.Friend => Ok(views.html.joiner.form.address(privateFields, marketingChoices, passwordExists))
         case paidTier =>
           val pageInfo = PageInfo.default.copy(stripePublicKey = Some(request.touchpointBackend.stripeService.publicKey))
-          val showSubscriberFields = tier == Tier.Partner
-          Ok(views.html.joiner.form.payment(paidTier, privateFields, marketingChoices, passwordExists, pageInfo, showSubscriberFields))
+          Ok(views.html.joiner.form.payment(paidTier, privateFields, marketingChoices, passwordExists, pageInfo))
```

Comment:
That was done as part of https://github.com/guardian/membership-frontend/commit/fca5d4f7d52d0b8e4b48a83d4a3289d05b3fca5c